keyboards/telophase/keymaps/default/keymap.c

```diff
@@ -0,0 +1,104 @@
+// this is the style you want to emulate.
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+
+#include "telophase.h"
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum telophase_layers
+{
+	_QWERTY,
+	_LOWER,
+	_RAISE,
+	_ADJUST,
+};
+
+enum telophase_keycodes 
+{
+  LOWER = SAFE_RANGE,
+  RAISE,
+  ADJUST
+};
+
+#define LONGPRESS_DELAY 150
+#define LAYER_TOGGLE_DELAY 300
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[_QWERTY] = { /*QWERTY*/
```

Comment:
This should use the layout formatting here: 
```c
	[0] = LAYOUT_ortho_4x12(
		KC_ESC,    KC_Q,      KC_W,    KC_E,    KC_R,    KC_T,      KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,
		KC_TAB,    KC_A,      KC_S,    KC_D,    KC_F,    KC_G,      KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
		KC_LSFT,   KC_Z,      KC_X,    KC_C,    KC_V,    KC_B,      KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_ENT,
		KC_LCTL,   KC_LGUI,   KC_LALT, ADJUST,  LOWER,   KC_SPC,    KC_SPC,  RAISE,   KC_LEFT, KC_DOWN, KC_UP,   KC_RIGHT
  ),
```

Reply:
Try as I might, I could not make this work. It think it has to do with the fact that this board is wireless?

Quote the Telophase readme:

This keyboard uses a completely different 'matrix scan' system to other keyboards, it relies on an external nRF51822 microcontroller maintaining a matrix of keystates received from the keyboard halves.

Reply:
To be honest, the `LAYOUT` formatting shouldn't affect that.  It just re-formats what you have into a more human readable format.  But this MUST be supported by the keyboard (and the "h" file, specifically). 

For instance: https://github.com/qmk/qmk_firmware/blob/master/keyboards/mitosis/mitosis.h#L52

Reply:
Not sure what my original blocker was here, after your advice I feel I was able to achieve this.